src/components/color/index.js

```diff
@@ -116,7 +116,7 @@ color.clean = function(container) {
             if(!Array.isArray(el0) && el0 && typeof el0 === 'object') {
                 for(j = 0; j < val.length; j++) color.clean(val[j]);
             }
-        } else if(val && typeof val === 'object') color.clean(val);
+        } else if(val && typeof val === 'object' && !ArrayBuffer.isView(val)) color.clean(val);
```

Comment:
Let's only allow typed arrays here by using the `isTypedArray` function.
https://github.com/plotly/plotly.js/blob/65f739ddb75cac1e809d8a57098c52833dafd151/src/lib/array.js#L8-L11
To use that you could require it like this:
```js
var isTypedArray = require('../../lib/array').isTypedArray;
```